colearn_keras/keras_cifar10.py

```diff
@@ -35,6 +35,52 @@
 LABEL_FL = "labels.pickle"
 
 
+def _make_loader(images: np.array,
```

Comment:
The dataloader now needs to be registered before the model which references it, which is why the order has changed in all these example files.